include/uk/compiler.h

```diff
@@ -0,0 +1,195 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/**
+ * Copyright (c) 1991, 1993
+ *	The Regents of the University of California.  All rights reserved.
+ * Copyright (c) 2017, NEC Europe Ltd., NEC Corporation. All rights reserved.
+ * Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * You may not use this file except in compliance with the License.
+ */
+
+#ifndef _UK_COMPILER_H_
```

Comment:
One last change: Please change these to `__UK_COMPILER_H__` here and at the comment at the end of the file. Thanks!